Dockerfile

```diff
@@ -115,11 +108,12 @@ VOLUME  "${unitycatalog_home}"
 
 WORKDIR "$unitycatalog_home"
 
-# Copy the Uber-Jar from the previous stage
-COPY --from=assemble_server "${server_assembly_jar}" "${unitycatalog_jars}/"
+COPY --from=package_server "${server_jar}" "${unitycatalog_jars}/"
```

Comment:
It appears the `${server_jar}` isn't being used - did you mean to use `${jars_directory}` which is included in line 116?

Reply:
It is used indirectly when the server is started up at line 129 using `jars/*`.

This "feature" makes the build `assembly`-less and more matching what the regular non-docker build gives.

Reply:
Oh got it!